misc/docs-linter/go.mod

```diff
@@ -0,0 +1,19 @@
+module linter
+
+go 1.21.6
+
+require (
+	github.com/gnolang/gno v0.0.0-20240516161351-0c9849a8ef0c
+	github.com/stretchr/testify v1.9.0
+	golang.org/x/sync v0.7.0
+	mvdan.cc/xurls/v2 v2.5.0
+)
+
+require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/peterbourgon/ff/v3 v3.4.0 // indirect
```

Comment:
Why do we need to use this package for a single flag?
We can use the standard go flag library for the lint commands

Reply:
I am using our commands package (`github.com/gnolang/gno/tm2/pkg/commands`) and I think `ff/v3` is a dependency on it. I would keep it in in the possibility that we add more flags later, seems like there is no harm in this.